buildSrc/src/main/kotlin/mockito.root.releasing-conventions.gradle.kts

```diff
@@ -0,0 +1,73 @@
+import gradle.kotlin.dsl.accessors._edf0c2271024fd9311959755ab251c7b.ext
+
+plugins {
+    id("org.shipkit.shipkit-auto-version")
+    id("org.shipkit.shipkit-changelog")
+    id("org.shipkit.shipkit-github-release")
+
+    id("io.github.gradle-nexus.publish-plugin")
+}
+
+val isSnapshot = version.toString().endsWith("SNAPSHOT")
+tasks {
+    generateChangelog {
+        githubToken = providers.environmentVariable("GITHUB_TOKEN").get()
+        previousRevision = project.ext["shipkit-auto-version.previous-tag"].toString()
```

Comment:
In a follow-up PR, let's refactor this so that we also use an environment variable for this.